package.json

```diff
@@ -23,6 +23,7 @@
     "backbone-computedfields": "0.0.10",
     "backbone.marionette": "^2.4.1",
     "backbone.stickit": "^0.9.2",
+    "bootstrap": "^3.3.5",
```

Comment:
did you check of this is bringing conflicts with sites that already include bootstrap ? or some other version of bootstrap